local_planner/launch/example.launch

```diff
@@ -1,36 +1,28 @@
 <launch>
-
-
-    <arg name="mavros_transformation" default="0" />
-    
     <param name="use_sim_time" value="false" />
 
-    <node pkg="tf" type="static_transform_publisher" name="tf_90_deg"
-          args="0 0 0 $(arg mavros_transformation) 0 0 world local_origin 10"/>
+    <!-- Transformation between the fcu and camera frame. It needs to be changed
+    according to the specific setup. Please measure the dispacement between the
+    camera and the flight controller in ENU body frame and update the x, y, z,
+    yaw, pitch, and roll in meters-->
```

Comment:
So a question I might ask here is which are the x, y, z etc params. (ie you need to confirm that they are the `args=` and ordered as you say.